libra/src/command/config.rs

```diff
@@ -0,0 +1,177 @@
+use clap::Parser;
+
+use crate::internal::config;
+
+#[derive(Parser, Debug)]
+pub struct ConfigArgs {
+    /// Add a configuration entry to database
+    #[clap(long, group("mode"), requires("valuepattern"))]
+    add: bool,
+    /// Get a single configuration entry that satisfied key and value pattern from database
+    #[clap(long, group("mode"))]
+    get: bool,
+    /// Get all configuration entries that satisfied key and value pattern from database
+    #[clap(long("get-all"), group("mode"))]
+    get_all: bool,
+    /// Remove a single configuration entry from database
+    #[clap(long, group("mode"))]
+    unset: bool,
+    /// Remove all the configuration entries that satisfied key and valuepattern from database
+    #[clap(long("unset-all"), group("mode"))]
+    unset_all: bool,
+    /// List all the configuration entries from database
+    #[clap(long, short, group("mode"))]
+    list: bool,
```

Comment:
Consider breaking down the operations (`add`, `get`, `unset`, etc.) into separate **sub-commands** rather than **args** (sth. like `--add`).
You can refer to the implementation of  [`RemoteCmds`](https://github.com/web3infra-foundation/mega/blob/ee47daa2d6a9fa2044772fcc84a5b6e8039923ed/libra/src/command/remote.rs#L5).

Reply:
I can't quite understand why using subcommands would be better here. I've read most of the command code and found that most of them don't use subcommands. Is it because subcommands are more intuitive and easier to understand?